router/internal/trace/config.go

```diff
@@ -1,41 +1,52 @@
 package trace
 
-import "time"
+import (
+	"time"
+
+	"github.com/wundergraph/cosmo/router/internal/otel/otelconfig"
+)
 
 // ServerName Default resource name.
 const ServerName = "cosmo-router"
 
-// Config represents the configuration for the agent.
-type Config struct {
-	Enabled bool
-	// Name represents the service name for tracing. The default value is wundergraph-cosmo-router.
-	Name     string
+type Exporter struct {
 	Endpoint string
-	// Sampler represents the sampler for tracing. The default value is 1.
-	Sampler       float64
-	Batcher       KindOtlp
+
+	Exporter      otelconfig.Exporter
 	BatchTimeout  time.Duration
 	ExportTimeout time.Duration
-	// OtlpHeaders represents the headers for HTTP transport.
+	// Headers represents the headers for HTTP transport.
 	// For example:
 	//  Authorization: 'Bearer <token>'
-	OtlpHeaders map[string]string
-	// OtlpHttpPath represents the path for OTLP HTTP transport.
+	Headers map[string]string
+	// HTTPPath represents the path for OTLP HTTP transport.
 	// For example
 	// /v1/traces
-	OtlpHttpPath string
+	HTTPPath string
+}
+
+// Config represents the configuration for the agent.
+type Config struct {
+	Enabled bool
+	// Name represents the service name for tracing. The default value is wundergraph-cosmo-router.
```

Comment:
It's just `cosmo-router`

Reply:
I just moved that around 😅 - but fixed, which is what matters!